### PR TITLE
closes #134 new contributors blog

### LINKED ISSUE #134: Blog Post:  Tips for first time open source contributors
### Blog Post

Tips for first time open source contributors

### PR DESCRIPTION
**Pull request by rossfarrugia**
New blog around tips for new Open source contributors. 

Needs @cpiraux approval before we merge/publish as I shared her story from https://github.com/pharmaverse/pharmaverse/issues/284. Celine - please feel free to add any comments!

### CODE REVIEW

File: posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+Memories like this stay with us, and they can have a habit of holding us
+back in later life. It never feels comfortable being the "new kid" and
+it takes a brave step to jump in and be open to the new connections and
+relationships that might come from this.
+
+Getting involved with open source can feel similarly daunting. You'll
+likely get hit with impostor syndrome making you doubt whether your code
+is worthy of sharing or you may fear being judged publicly by others.
+Even more fundamentally, you might lack certain skills or experience
+(such as git) which could make an initial barrier.
```
Comment (manciniedoardo) [inline posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd:45]: Add explanation for those not familiar with git

```suggestion
(such as [git](https://git-scm.com/book/en/v2/Getting-Started-What-is-Git%3F)) which could make an initial barrier.
```

File: posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+One of the key tenets when we started Pharmaverse was for this to be
+driven by a passionate community of people, many of whom that actually
+use the packages, as who better to empathize with the challenge at hand
+than the users themselves. So firstly, please erase from your mind the
```
Comment (manciniedoardo) [inline posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd:50]:
```suggestion
than the users themselves? So firstly, please erase from your mind the
```

File: posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+are made, and comparisons between commits and branches are performed. I
+also learned about the functioning of comments, pull requests, and the
+review process.*
+
+*Once his PR was completed, it was time for the review phase, during
+which I ran the new code to understand the changes and experiment with
+it. This step also involved two technical requirements: firstly, linking
+my GitHub account with RStudio to access the code in the branch, and
+secondly, understanding how to execute code within the branch. Since
+installing packages using 'install.packages' wouldn't function for
```
Comment (manciniedoardo) [inline posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd:137]:
```suggestion
installing packages using `install.packages()` wouldn't function for
```

File: posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+development code, it was essential to know about the function
+'devtools::load_all()' to execute the code.*
```
Comment (manciniedoardo) [inline posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd:139]:
```suggestion
`devtools::load_all()` to execute the code.*
```

File: posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+*As I became familiar with both the package and GitHub, I felt more
+confident to start my first implementation. I read the book 'R Packages'
+to gain a better understanding of R package development. The first
+chapter, entitled 'The Whole Game', provided a good overview of R
+package development and introduced useful functions such as
+'load_all()', 'document()', 'test()', and 'check()'.*
```
Comment (manciniedoardo) [inline posts/2024-03-11_tips_for__first_.../tips_for__first__time__contributors.qmd:146]:
```suggestion
`load_all()`, `document()`,  `test()`, and `check()`.*
```

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Great post! Just some minor things.

**Review by manciniedoardo — APPROVED** [review]

**Review by cpiraux — APPROVED** [review]
Great post! It looks good to me. Thank you, @rossfarrugia, for including my story.

**Comment by rossfarrugia** [comment]
Thanks Edoardo & Celine! happy for this to be merged anytime. I think the CI checks are failing for other blogs currently from memory, but shout if you want me to fix anything here.

**Comment by bms63** [comment]
Hi all,

New spelling and link issues popped up. I have fixed them.  

I only had a few minutes for this today...I love this post and nice to have someting like this on our blog.  With any post I get a little sad when it is just a  "Wall of Text".  Any chance we could break it up with some pretty images???  

Just a couple!!

**Comment by rossfarrugia** [comment]
> Any chance we could break it up with some pretty images???

I added one for you at the end @bms63 - i tried out adding a few in between the texts but actually felt it had a negative effect reading by breaking up the sections for this one as its so short that i feel it flows nicely as it is. feel free to merge/publish and share some comms if you're OK with it.

I'm presenting at an upcoming PHUSE Webinar Wednesday next month and will reference this blog.

**Comment by bms63** [comment]
> > Any chance we could break it up with some pretty images???
> 
> I added one for you at the end @bms63 - i tried out adding a few in between the texts but actually felt it had a negative effect reading by breaking up the sections for this one as its so short that i feel it flows nicely as it is. feel free to merge/publish and share some comms if you're OK with it.
> 
> I'm presenting at an upcoming PHUSE Webinar Wednesday next month and will reference this blog.

Okay makes sense!! :)

**Pull request merged**